connect/runtime/src/test/java/org/apache/kafka/connect/runtime/WorkerSinkTaskThreadedTest.java

```diff
@@ -147,7 +150,6 @@ public void testPollsInBackground() throws Exception {
             workerTask.iteration();
         }
         workerTask.stop();
-        workerTask.awaitStop(Long.MAX_VALUE);
```

Comment:
This looks like fallout from the refactoring of WorkerTask and how its lifecycle is managed, but this removal also highlights that this test is kind of misnamed now -- it doesn't actually have any extra threads anymore, right? Especially since the class was refactored as Runnable instead of a thread?